src/reaper_osara.cpp

```diff
@@ -1298,7 +1302,19 @@ LRESULT CALLBACK keyboardHookProc(int code, WPARAM wParam, LPARAM lParam) {
 	} else if (wParam == VK_F6 && !(lParam & 0x80000000)) {
 		if (maybeSwitchToFxPluginWindow())
 			return 1;
-	}
+	}else if(wParam == VK_HOME || wParam == VK_END || wParam == VK_SHIFT){
```

Comment:
Is it useful to pass shift at the moment given that it's not working as expected?

Reply:
It might be something I forgot to delete after failing to make the "select to end" keystroke work. Currently, it is not necessary as you pointed out.